Libplanet/Net/Swarm.cs

```diff
@@ -256,7 +270,7 @@ public void CopyTo(Peer[] array, int arrayIndex)
 
                 _dealers.Clear();
 
-                Running = false;
+                _runningEvent = new TaskCompletionSource<object>();
```

Comment:
It seems to be able to use `setter` of `Running` to eliminate duplicates. (it's trivial.)

Reply:
👌